Initial attempt at scala automated comments for two-fer (#1148)
exercism/website-copy: automated-comments/scala/general/proper_function_name.md

> The function name should match exactly what was used in the tests.

Reviewer: I would turn these around:

```suggestion
Rename the function to `%{functionName}`.
The function name should match exactly what is used in the tests.
```

> Rename the function to `%<functionName>s`.

Reviewer: ```suggestion
Rename the function to `%{functionName}`.
```

Don't apply this if you apply the suggestion above.

Rename the function to `%{functionName}`.
The function name should match exactly what is used in the tests.

> Rename the function to `%<functionName>s`.

Reviewer 2: Isn't this a duplicate of the first line?

Reviewer: It is, it was moved (reversed order) and this suggestion was not applied (remove this line).

```suggestion
```
